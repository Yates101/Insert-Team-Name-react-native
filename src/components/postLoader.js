import React, { useState } from 'react';
import { StyleSheet, View, Text } from 'react-native';

const PostLoader = () => {
const [postData, setPostData] = useState('')
console.log(postData)
async function PostFetch() {
  
    fetch("https://acebook--backend.herokuapp.com/posts")
    .then((response) => response.json())
      .then((json) => setPostData(json))
      console.log(postData)
      .catch((error) => console.error(error))

  }
    return(
      <View>
        <Text>{postData}</Text>
      </View>
  )
};

export {PostLoader};