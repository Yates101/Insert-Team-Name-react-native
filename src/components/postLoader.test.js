import 'react-native';
import React from 'react';

import {postLoader} from './postLoader';
import renderer from 'react-test-renderer';
import '@testing-library/jest-native/extend-expect';

describe("<postLoader />", () => {
  const tree = renderer.create(<postLoader />).toJSON();
  console.log(tree)
  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  })
})