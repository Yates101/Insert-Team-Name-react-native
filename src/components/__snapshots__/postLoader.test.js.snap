// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<postLoader /> renders correctly 1`] = `<postLoader />`;
